site/jekyll/getting-started/tutorial.md

```diff
@@ -402,13 +402,16 @@ namespace ReactDemo.Controllers
 Let's also add a new controller action to return the list of comments:
 
 ```csharp
+[OutputCache(Location = OutputCacheLocation.None)]
 public ActionResult Comments()
 {
 	return Json(_comments, JsonRequestBehavior.AllowGet);
 }
 ```
 
-And a corresponding route in `App_Start\RouteConfig.cs`:
+The `OutputCache` attribute is used here to prevent IE from caching the ajax request. IE tries to optimize things by assuming that identical requests will return identical responses. Subsequent calls to the Comments action will simply return the cached response from the first call, the result being that the comment list is never updated in IE as new comments are added. When designing a real world API, cache times of API requests should be considered more carefully. For this tutorial it is easiest to simply disable caching.
+
+Finally we add a corresponding route in `App_Start\RouteConfig.cs` to simplify the request URL a bit:
```

Comment:
> Why shouldn't a production app use the default route?

It does work fine and you can do it that way, but ideally you should think about a good URL structure for pages in the app. Using the default route exposes internal implementation details (controller and action names) that you might want to change in the future. [Cool URIs don't change](http://www.w3.org/Provider/Style/URI.html), and well-designed URIs should remain the same regardless of the code behind them or the backend technology being used. Removing the default route makes you think about every URI as you build the site and generally results in cleaner more maintainable URIs.

I think at least some of that is personal preference though :P